### ISSUE TITLE
Failing test: `tests/core/creators/aria`

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#5189]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run test on IE11

### Expected result

*What is the expected result of the above steps?*

Test is passing.

### Actual result

*What is the actual result of the above steps?*

Test fails with: 

Editor's title label is incorrect
Expected: &nbsp; (string)
Actual: &amp;nbsp; (string)

## Other details

* Browser: IE11
* OS: Win10
* CKEditor version: 4.19.0 (pre release)
* Installed CKEditor plugins: development

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5193
